specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/applicationGateway.json

```diff
@@ -1589,7 +1589,12 @@
         "provisioningState": {
           "type": "string",
           "description": "Provisioning state of the backend http settings resource. Possible values are: 'Updating', 'Deleting', and 'Failed'."
-        }
+        },
+         "port": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Custom port which will be used for probing the backend servers. The valid value ranges from 1 to 65535. In case not set, port from http settings will be used. This propery is valid for Standard_v2 and WAF_v2 only."
```

Comment:
Since it has a specific range, should it have the minimum/maximum constrains set on the API? EX: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/preview/2017-08-01-beta/analysisservices.json#L879-L880